sormas-backend/src/main/java/de/symeda/sormas/backend/survey/SurveyFacadeEjb.java

```diff
@@ -82,6 +103,76 @@ public SurveyDto save(@Valid SurveyDto dto) {
 		return toDto(survey);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._SURVEY_EDIT)
+	public void uploadDocumentTemplate(SurveyReferenceDto surveyRef, DocumentTemplateDto uploadedDocumentTemplate, byte[] fileContent)
+		throws DocumentTemplateException {
+		Survey existingSurvey = surveyRef.getUuid() != null ? surveyService.getByUuid(surveyRef.getUuid()) : null;
```

Comment:
Survey is required, just add @Notnull annotation on the argument and remove the null checks and assertion

Reply:
made the changes